packages/wc-compass/src/layouts/about/_about.scss

```diff
@@ -0,0 +1,81 @@
+div.cdg-about-section {
+  display: flex;
+  flex-direction: row;
+  color: var(--grey100);
+  padding-top: 32px;
+  padding-bottom: 32px;
+  container-name: CdgAbout;
+  container-type: inline-size;
+  justify-content: space-between;
+
+  .cdg-about-left {
```

Comment:
...-left-content or ...left-col or even ... left-column for meaningful class name

Reply:
fixed